apps/bestofjs-nextjs/src/app/projects/[slug]/project-details-npm/dependencies-section.tsx

```diff
@@ -39,26 +39,11 @@ export async function DependenciesSection({
       <CollapsibleContent className="space-y-4 py-4">
         {projects.length > 0 && (
           <Card>
-            <CardHeader>
```

Comment:
I think we can remove the `CardHeader` from the imports now.

Reply:
Good catch, I will set up the ESLint rule that should not allow unused variables in a separate PR.